### PR TITLE
Add more description about how to upgrade external dependencies' version according to spring-boot-dependencies and spring-cloud-dependencies.

### PR DESCRIPTION
**Pull request by chenrujun**
As title.

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @saragluna , please help to review this PR when you have time.

File: sdk/spring/scripts/README.md [hunk 1]
```diff
@@ -4,12 +4,14 @@
...
-2. Run command `python .\sdk\spring\scripts\get_spring_boot_managed_external_dependencies.py`.
-3. Update `SPRING_BOOT_VERSION` in `sync_external_dependencies.py`.
-4. Run command `python .\sdk\spring\scripts\sync_external_dependencies.py`.
-5. Run command `python .\eng\versioning\update_versions.py --ut external_dependency --sr`
-6. Run command `.\eng\versioning\pom_file_version_scanner.ps1 -Debug`. If there is error, fix it.
-7. Update changelog about compatible Spring Boot versions and Spring Cloud versions in `/sdk/spring/CHANGELOG.md`.
+## Upgrade external dependencies' version according to spring-boot-dependencies and spring-cloud-dependencies.
+
+1. Get `SPRING_BOOT_VERSION` from [spring-boot-dependencies tags](https://github.com/spring-projects/spring-boot/tags). Get `SPRING_CLOUD_VERSION` from [spring-boot-dependencies tags](https://github.com/spring-cloud/spring-cloud-release/tags). Note that spring-cloud version should compatible with spring-boot version. Refs: [Spring Cloud Release train Spring Boot compatibility](https://spring.io/projects/spring-cloud).
```
Comment (saragluna) [inline sdk/spring/scripts/README.md:9]: Need change:  Get `SPRING_CLOUD_VERSION` from [spring-boot-dependencies tags]

**Review by saragluna — CHANGES_REQUESTED** [review]

File: sdk/spring/scripts/README.md (hunk 1 above)
Comment (chenrujun) [reply sdk/spring/scripts/README.md]: Updated.

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**